noir/noir-repo/tooling/profiler/src/cli/gates_flamegraph_cmd.rs

```diff
@@ -97,7 +103,7 @@ fn run_with_provider<Provider: GatesProvider, Generator: FlamegraphGenerator>(
             &debug_artifact,
             artifact_path.to_str().unwrap(),
             &func_name,
-            &Path::new(&output_path).join(Path::new(&format!("{}_gates.svg", &func_name))),
```

Comment:
Better to compose function name with output filename, so for artifacts that have multiple functions (#[fold]) we don't overwrite the same file over and over. You can also change the default value to something other than main